### DISCUSSION TITLE
2023/05/16/%E8%80%83%E5%90%8E%E5%8F%8D%E6%80%9D/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#10] [Announcements]
# 2023/05/16/%E8%80%83%E5%90%8E%E5%8F%8D%E6%80%9D/

It's Florance!

https://floranceyeh.netlify.app/2023/05/16/%E8%80%83%E5%90%8E%E5%8F%8D%E6%80%9D/

**Comment by FloranceYeh** [comment]
只是生气罢了，平常的我还是很和蔼的

Florance 从来不讲脏话的啦

=）
